uninstaller/uninstall-helper.ps1

```diff
@@ -4,6 +4,22 @@ for ($i=0; $i -le $args.Count - 1; $i++) {
 }
 $timeStamp = $args[-1]
 
+$subfolders = Get-ChildItem "$folder\.." -Directory -ErrorAction SilentlyContinue | ForEach-Object { $_.FullName }
```

Comment:
Would be good if we can iterate over each folder and identify what is exactly locked.